code/style.css

```diff
@@ -1,50 +1,75 @@
 body {
   margin: 0;
   font-family: "Courier New", Courier, monospace;
+  background-image: url("./assets/crime.jpg");
+  background-size: cover;
```

Comment:
A suggestion: now when you are on bigger screens (maybe you haven't looked at the page on larger screens) the background image repeats it self. If you add props `background-repeat: no-repeat;` it will not be repeated. But then you get a white background where the image doesn't fit. Then you can add `background-size: cover;` to make it go all over the `body`. And if you want the `body` to be 100% of the height you can add `height: 100vh;`.

Before:
![image](https://user-images.githubusercontent.com/22143776/68029919-94ac1b80-fcb8-11e9-844c-a07908d35c7c.png)


After:
![image](https://user-images.githubusercontent.com/22143776/68029896-84943c00-fcb8-11e9-8bf3-c328f2540b39.png)